### DISCUSSION TITLE
Can't get RoR to display during roast

### DISCUSSION THREAD

**Discussion by N6GQ** [artisan-roaster-scope/artisan#767] [General]
Apologies for probably a stupid question but I'm not having any luck resolving this myself, looking to you for help. I'm an artisan n00b and have reviewed the docs, faq, questions, etc. and still not having any luck. This is Artisan 2.4.6 with Phidget RTDs for ET/BT/MET. That all works wonderfully. I do have a DeltaBT LCD that appears to work properly during the roast, but I cannot get the curve to display while roasting. Sometimes it will show up after the roast. Sometimes not. I am pressing the "CHARGE" button when charging, however the charge and all the other buttons are grey, and they don't put any  markers on the roast graph. So I'm guessing this is part of my problem somehow but I don't know how yet.

I can show settings but I'm not sure whats relevant. I did try the configuration option in Axes to auto-fit the DeltaBT curve based on a bug I saw from an earlier version but that doesn't help either.

In short, what should I look for, and what settings should I post here for criticism?

Thanks kindly,

Jeff in Texas

**Comment by MAKOMO** [comment]
This is an FAQ: [ I can not see the rate-of-rise curve during roasting?](https://artisan-scope.org/help/#FAQ-Recording-7)

EDIT: link fixed

**Reply by N6GQ** [reply]
Yep, I had read that, and waited for the period after "start" - the charge button turns blue for a second then turns grey with the rest of the buttons. It seems its auto-charging yet as far as I can tell auto-charge is turned off.

**Reply by N6GQ** [reply]
Ok,, I think I found the problem - auto-alarms and auto-charge, turning it all off seems to have fixed it...thanks!

**Reply by MeanCatCoffee** [reply]
Makomo, 

I'm getting a  404 error from your link. I'm having the same issue.

**Reply by poundy** [reply]
The link should be to the FAQs at https://artisan-scope.org/help/#FAQ 

The way I interpret the FAQ there, it implies that if you haven't actually started your roasting correctly, the ROR calc and graph won't appear, even though the ET/BT lines will track on the roast.  Critically, the CHARGE is the required event.
